src/js/angular/ttyg/templates/chat-item-detail.html

```diff
@@ -1,16 +1,16 @@
 <link href="css/ttyg/chat-item-details.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
 
 <div class="chat-detail">
-    <div class="user-message">
+    <div class="user-message" ng-if="chatItemDetail.question">
```

Comment:
For now this will do the job, but at some point we have to extend the model and avoid these checks here.